README.md

```diff
@@ -1,3 +1,11 @@
 ### sample_infra_terraform
 
```

Comment:
Update the ReadMe to include details in the following format:
  - Overview (A one or two sentences of what the goal of the sample project is)
  - High level workflow 
       - What is used for each adapter (Source Control Provider, Language, Notification, Services/Tools, Docker Registry, Container Service and IaaS/PaaS)
       - Is a standard or custom image used
       - What really happens in this sample project
  - Prerequisites/Requirements: (Example: Env Variables required to ensure the sample project works successfully)
  - Instructions on using the sample project (Include step-wise instructions on installing, configuring, executing the project)
  - Instructions for setting up 3rd party services (optional)
     - If EC2 or Google clusters, etc have to be set up, instructions on setting them up need to be written
  - Resources
    - Relevant Shippable docs, blogs or 3rd party sites that give additional information